admin_manual/configuration_files/external_storage_configuration_gui.rst

```diff
@@ -131,6 +131,19 @@ on your network!
 .. figure:: external_storage/images/user_mounts.png
    :alt: Checkboxes to allow users to mount external storage services.
 
+Setting Up Google Drive and Dropbox Connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When an external storage is created which uses either Google Drive or Dropbox, a link to the respective configuration page is available, next to the service name. 
+
+.. figure:: external_storage/images/external-storage-google-drive-and-dropbox-configuration.png
+   :alt: Links to Google Drive and Dropbox app configuration pages.
+
+In the screenshot above, you can see that two external storage connections have been created, but not configured. 
+One goes to Google Drive, the other to Dropbox. 
+If you click the cog icon next to the name of either, the respective app configuration page will open in a new tab, or a new window. 
+From there, you can manage the configuration and obtain the respective credentials needed for configuring the connection.
```

Comment:
What you have written here is fine, as far as it goes. So you could commit this and then think about whether to add to it.
The challenge is that whenever Dropbox or... third-party changes their UI/workflow then the oC doc becomes stale.